cms/search/checks.py

```diff
@@ -0,0 +1,44 @@
+from django.conf import settings
+from django.core.checks import Error, register
+
+
+@register()
+def check_kafka_settings(app_configs, **kwargs) -> list[Error]:
+    """Check that required Kafka settings are present."""
+    errors = []
+
+    kafka_settings = [
```

Comment:
these should only kick in if the provider is "kafka"

Reply:
Done!